test/_init_tests.js

```diff
@@ -160,6 +166,10 @@ class TestConfig {
     return this.if(this.platform.isIos.bind(this.platform));
   }
 
+  ifIe() {
+    return this.if(this.platform.isIe.bind(this.platform));
```

Comment:
Isn't this just a function that can be used by a test writer to run a test exclusively on IE, should they choose to? The way I understood this class, the default for `describe()` is to run tests on all platforms except those on which we call `skipFoo`, and the behavior for `ifFoo` is to run that test only on `Foo` browser. 

I'm still testing this out, so I could be wrong.